### DISCUSSION TITLE
possible way to track when user click close app on window?

### DISCUSSION THREAD

**Discussion by kechankrisna** [halildurmus/win32#327] [Q&A]
this feature will allow developer to do something when app closed on windows. Thank you in advance.

**Comment by Aditya-Vishwa** [comment]
I can't understand your question can you please elaborate?

**Comment by loic-sharma** [comment]
This should already be possible by handling the `WM_CLOSE` window message. See:

* Example app that handles `WM_CLOSE`: https://github.com/timsneath/win32/blob/6f1b25c74f8a56fa62ce0c463f2d70be33feb822/example/notepad/notepad.dart#L224-L228
* `WM_CLOSE` docs: https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-close
